hack/install.sh

```diff
@@ -440,6 +440,11 @@ do_install() {
 				( set -x; $sh_c 'sleep 3; apt-get install -y -q curl ca-certificates' )
 				curl='curl -sSL'
 			fi
+			if [ ! -e /usr/bin/gpg ]; then
+				apt_get_update
+				( set -x; $sh_c 'sleep 3; apt-get install -y -q gnupg' )
```

Comment:
In stetch and later, `gunpg2` is the preferred implementation, so I think it's probably prudent to try installing that one first and only install `gnupg` if that install fails